### PR TITLE
feat(KUI-1260): upgrade to KTH Style 10

### PR DESCRIPTION
**Pull request by karlandindrakryggen**
Main goal of this PR is to upgrade KTH Style to version 10. 

A short summary of what have been done:
* Upgrading KTH packages and adding new `@kth/style` along side with old `kth-style` (the old will be deprecated)
* General cleanup of DOM structure to algin with other Om kursen apps. 
* New header and footer
* New of style of all Buttons, Alerts and SideMenu
* Added "prose" style to articles/text sections
* General cleanup of style including font-family, colors and more.
* Move of toolbar url (Personliga menyn) to parameters file to support ref version in ref/loaclhost

Some code are duplicated a cross different Om kursen repos. Such SCSS has been moved to `shared.scss` and components to `/shared-components`. PRs for other Om kursen apps that are upgraded at the same time with same structure:
* Kopps public: https://github.com/KTH/kopps-public-react-web/pull/242
* Kursinfo: https://github.com/KTH/kursinfo-web/pull/328
* Kurs PM: https://github.com/KTH/kurs-pm-web/pull/373

### CODE REVIEW

File: public/css/_shared.scss [hunk 1]
```diff
@@ -0,0 +1,150 @@
+// This file containt style shared between the four public facing Om kursen apps (kopps-public-react-web, kursinfo-web, kurs-pm-web and kursutveckling-web)
```
Comment (belanglos) [inline public/css/_shared.scss:1]: s/containt/contains ;-)

File: public/js/app/components/AnalysisTable.jsx [hunk 2]
```diff
@@ -10,14 +10,9 @@ const ActiveOrDisabledLink = ({ fileName, linkTitle, storageUri, roundName, vali
   return (
     <p>
       {fileName === '' ? (
-        // eslint-disable-next-line jsx-a11y/anchor-is-valid
```
Comment (belanglos) [inline public/js/app/components/AnalysisTable.jsx:13]: Does this make a difference for screen readers? I.e. to replace a disabled link element with a span?

File: public/js/app/components/DocumentLinksNav.jsx [hunk 3]
```diff
@@ -9,16 +9,11 @@ import LinkToValidSyllabusPdf from './LinkToValidSyllabus'
 const ActiveOrDisabledPdfLink = ({ ariaLabel, href = '', className = '', linkTitle, translate, validFrom = '' }) => {
   const { no_added_doc } = translate
   return (
-    <p key={linkTitle}>
+    <>
       {href === '' ? (
-        // eslint-disable-next-line jsx-a11y/anchor-is-valid
```
Comment (belanglos) [inline public/js/app/components/DocumentLinksNav.jsx:14]: Same question here, reg. disabled link

**Review by belanglos — APPROVED** [review]
Good stuff!

Added two comments about disabled links, could have done research, but wanted to check first whether you've already thought about this. :)

File: public/js/app/components/AnalysisTable.jsx (hunk 2 above)
Comment (karlandindrakryggen) [reply public/js/app/components/AnalysisTable.jsx:13]: This hopefully makes better for a screen reader. A `<a>` element is meant for navigation and should always have a href. So a `<a>` without a href (or a "disabled" `<a>`) makes no sense imo. 

An alternative would be to use a `<button>` in this case (that actually can be disabled), but I think that might bit confusing i other ways. 

Some background to the eslint rule (that previously was disabled for the line):
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/anchor-is-valid.md

File: public/css/_shared.scss (hunk 1 above)
Comment (karlandindrakryggen) [reply public/css/_shared.scss]: fixed :)

File: public/js/app/components/AnalysisTable.jsx (hunk 2 above)
Comment (belanglos) [reply public/js/app/components/AnalysisTable.jsx:13]: Thanks for teaching me something! 🙏

**Review by belanglos — APPROVED** [review]

**Pull request merged**